### ISSUE TITLE
"error: Package ‘zoom-frm-20170125.1953’ is unavailable" during install

### ISSUE THREAD

**Issue by e-monson** [thefrontside/frontmacs#115]
```
Warning (initialization): An error occurred while loading ‘/home/pholisma/.emacs.d/init.el’:

error: Package ‘zoom-frm-20170125.1953’ is unavailable

To ensure normal operation, you should investigate and remove the
cause of the error in your initialization file.  Start Emacs with
the ‘--debug-init’ option to view a complete error backtrace.
```
I'm running Emacs 25.3.2 on Ubuntu 14.04.

**Comment by cowboyd** [comment]
Can you post the contents of your `init.el`?

**Comment by e-monson** [comment]
```
;; boot frontmacs

;; Added by Package.el.  This must come before configurations of
;; installed packages.  Don't delete this line.  If you don't want it,
;; just comment it out by adding a semicolon to the start of the line.
;; You may delete these explanatory comments.
(package-initialize)

(load (expand-file-name "init-frontmacs.el" user-emacs-directory))
```

**Comment by theMultitude** [comment]
I am getting the same error when first initializing.  Any word?

**Comment by Robdel12** [comment]
I can replicate this by setting up with an empty `.emacs.d` (Emacs 25.1.1) with frontmacs: 
![image](https://user-images.githubusercontent.com/2072894/35713256-028054d8-078c-11e8-8fa5-6d4ad192d2e7.png)

**Comment by Robdel12** [comment]
Related to: https://github.com/melpa/melpa/pull/5008 (zoom-frm was a wiki package)

**Comment by Robdel12** [comment]
This was released as `v0.2.18`! Let us know if there are anymore issues :)
